libbinlogevents/include/table_id.h

```diff
@@ -39,6 +39,7 @@ class Table_id
 public:
   Table_id() : m_id(0) {}
   explicit Table_id(unsigned long long id) : m_id(id) {}
+  Table_id(const Table_id& tid) { m_id = tid.m_id; }
```

Comment:
Is this necessary to get rid of a warning? Otherwise auto-generated copy constructor looks fine.

Reply:
It fixes gcc-9 warning:
```
/data/mysql-server/5.7/libbinlogevents/include/table_id.h:64:12: error: implicitly-declared ‘constexpr Table_id::Table_id(const Table_id&)’ is deprecated [-Werror=deprecated-copy]
```